packages/client/package.json

```diff
@@ -83,17 +83,17 @@
     "openfin-notifications": "1.23.0",
     "polished": "^4.1.1",
     "query-string": "^7.1.3",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
```

Comment:
19 should be out sometime this year.
Interested to see what's new and shiny

Reply:
yep, RT will be there as soon as a stable v19 comes out .. they were explicitly advising against bumping to 19 for the pre-releases unless you are a library owner or similar